libs/cypress/src/lib/commands/vega-wallet-receive-fauceted-asset.ts

```diff
@@ -23,10 +23,26 @@ export function addVegaWalletReceiveFaucetedAsset() {
       // @ts-ignore - ignoring Cypress type error which gets resolved when Cypress uses the command
       cy.get_asset_information().then((assets) => {
         const asset = assets[assetName];
-        for (let i = 0; i < asset.decimals; i++) amount += '0';
-        cy.exec(
-          `curl -X POST -d '{"amount": "${amount}", "asset": "${asset.id}", "party": "${vegaWalletPublicKey}"}' -u "hedgehogandvega:hiccup" http://localhost:1790/api/v1/mint`
-        );
+        if (assets[assetName] !== undefined) {
+          for (let i = 0; i < asset.decimals; i++) amount += '0';
+          cy.exec(
+            `curl -X POST -d '{"amount": "${amount}", "asset": "${asset.id}", "party": "${vegaWalletPublicKey}"}' -u "hedgehogandvega:hiccup" http://localhost:1790/api/v1/mint`
```

Comment:
chore (blocking): can we make the wallet user name here match what is setup in CI?

Reply:
have dropped the -u flag from curl command since no longer required